numpy/polynomial/polynomial.py

```diff
@@ -508,14 +509,14 @@ def polyder(c, m=1, scl=1, axis=0):
     Examples
     --------
     >>> from numpy.polynomial import polynomial as P
-    >>> c = (1,2,3,4) # 1 + 2x + 3x**2 + 4x**3
+    >>> c = (1, 2, 3, 4) # 1 + 2x + 3x**2 + 4x**3
     >>> P.polyder(c) # (d/dx)(c) = 2 + 6x + 12x**2
     array([  2.,   6.,  12.])
-    >>> P.polyder(c,3) # (d**3/dx**3)(c) = 24
+    >>> P.polyder(c, 3) # (d**3/dx**3)(c) = 24
     array([24.])
-    >>> P.polyder(c,scl=-1) # (d/d(-x))(c) = -2 - 6x - 12x**2
+    >>> P.polyder(c, scl=-1) # (d/d(-x))(c) = -2 - 6x - 12x**2
     array([ -2.,  -6., -12.])
-    >>> P.polyder(c,2,-1) # (d**2/d(-x)**2)(c) = 6 + 24x
+    >>> P.polyder(c, 2, -1) # (d**2/d(-x)**2)(c) = 6 + 24x
     array([  6.,  24.])
```

Comment:
I would be in favor of removing these comments. I don't think they provide much value besides the output.

Reply:
Maybe remove the right hand side; that is pretty obvious from the output. The left hand side may be helpful to confirm the reader's understanding of the arguments.